README.md

```diff
@@ -1,2 +1,25 @@
 # getting-started-with-rp_ap
 Sample tests to show how to use ReportPortalIntegration and ApplitoolsIntegration utilities
+
+## Build
+`mvn clean install -DskipTests`
+> If facing issues with dependencies not being resolved from https://jitpack.io, then check the `settings.xml` file you're using for building your maven projects. If you've proxies configured in the same, then make sure `jitpack.io` is part of `nonProxyHosts` configuration. For instance
+```xml
+     <proxy>
+         <id>httpmyproxy</id>
+         <active>true</active>
+         <protocol>http</protocol>
+         <host>someHost</host>
+         <port>8080</port>
+         <username>UserName</username>
+         <password>Password</password>
+         <nonProxyHosts>*.google.com|*jitpack.io</nonProxyHosts>
+     </proxy>
+```
+
```

Comment:
Add the necessary pre-requisites to run the tests.
ex: APPLITOOLS license key, and env var, device connected (for app tests), other software required to run the tests, etc.